### DISCUSSION TITLE
login credential support

### DISCUSSION THREAD

**Discussion by dkerr64** [nVuln/homebridge-lg-thinq#17] [Q&A]
Just got LG washer and dryer and installed the LG ThinQ mobile app.  I was given the option of using my AppleID to register and sign in, which I did so as to avoid yet-another userid/password.  So now the question... is it possible to use this plugin when I registered with LG app using AppleID?  I'm not sure if I have the option to change back to a LG userid/password.

Thanks.

**Comment by nVuln** [comment]
at this time this plugin only support login by LG account )maybe in future I'll added 3rd-party login)
for now, you can register new LG account (by username/password) and re-link your device, it's must easier
